week3/homework/src/actions.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const { readFile, writeFile } = require('fs');
+const { promisify } = require('util');
+const uuid = require('uuid/v4');
+const reading = promisify(readFile);
+const writing = promisify(writeFile);
+const TODO_FILE = 'todolist.json';
+
+const readTodos = () => {
+    return reading(TODO_FILE, 'utf8').then(JSON.parse).catch(() => ([]));
+};
+const toWrite = (todo) => {
+    return writing(TODO_FILE, JSON.stringify(todo, null, 2));
+};
+
+const createTodo = async (todo) => {
+    if (!todo.description) throw new Error('Please add description to your todo');
+    const todoList = await readTodos();
+    const id = uuid();
+    todo.id = id;
+    todo.status = 'Todo needs action';
```

Comment:
Why are you initially writing a string to `status` but then changing it to a boolean? That's a inconsistent.